specification/appendix-A.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Appendix A: Included Utilities
+description: Information on OpenFeature ecosystem utilities
+sidebar_position: 4
+---
+
+# Appendix A: Included Utilities
+
+This document contains requirements for auxiliary utilities provided by the SDK, such as testing utilities.
+
+## In-memory provider
+
+> Language specific OpenFeature SDK implementations **SHOULD** expose an in-memory provider built into the SDK.
+
+The in-memory provider is intended to be used for testing, SDK consumers may use it for their use cases.
+Hence, the packaging, naming and access modifiers must be set appropriately.
+
+Given below are features this provider **MUST** support,
+
+> - Provider must be initiated with a pre-defined set of flags provided to a constructor
```

Comment:
```suggestion
- Provider must be initiated with a pre-defined set of flags provided to a constructor
```
I don't think we want those quotes?

Reply:
Ya, actually I think we should get rid of these ones, just keep the top one on "Language specific OpenFeature SDK implementations **SHOULD** expose an in-memory provider built into the SDK."

Reply:
Removed with the latest commit